src/Tasks/Backup/BackupJob.php

```diff
@@ -214,7 +214,13 @@ protected function dumpDatabases(): array
 
             $dbType = mb_strtolower(basename(str_replace('\\', '/', get_class($dbDumper))));
 
-            $dbName = $dbDumper instanceof Sqlite ? 'database' : $dbDumper->getDbName();
+            $dbName = $dbDumper->getDbName();
+
+            if ($dbDumper instanceof Sqlite) {
+                $_ = explode('.', basename($dbName));
```

Comment:
Could you make it more clear what happens here?

Reply:
Certainly overcomplicated this! Now the assumption is the sqlite database ends with `.sqlite`. We'll use the filename of the database as part of the backup filename.